src/Http/Http.Extensions/src/DisableHttpMetricsAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http.Metadata;
+
+namespace Microsoft.AspNetCore.Http;
+
+/// <summary>
+/// Specifies that HTTP request duration metrics is disabled for an endpoint.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
```

Comment:
```suggestion
[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Delegate)]
```

Do we want to support applying this attribute directly on a minimal API handler like so?

```csharp
app.MapGet("/", [DisableHttpMetrics] () => { });
```

Extension methods are the more common pattern for minimal APIs but some people still prefer the attribute.

Also, does this benefit from a `DebuggerDisplay` attribution?

Reply:
I believe `AttributeTargets.Parameter` is what would allow this.

I'm going to leave it off because the parameter target is missing from most attributes. If this changes, it can be added everyone at once.